src/nvim/terminal.c

```diff
@@ -154,6 +151,8 @@ static Map(int, int) *color_indexes;
 static int default_vt_fg, default_vt_bg;
 static VTermColor default_vt_bg_rgb;
 
+// init/teardown {{{
```

Comment:
Such sections are handy when splitting source between files will produce too much code and you still want to have clear organization: I put them in `src/nvim/eval/typval.c`. I would not use `{{{` and `}}}` though, `{{{N` variant is easier to use.